local-volume/provisioner/pkg/discovery/discovery.go

```diff
@@ -111,6 +111,13 @@ func (d *Discoverer) discoverVolumesAtPath(class string, config common.MountConf
 		return
 	}
 
+	// Cache mount points to iterate through discovered paths (aka files) below
```

Comment:
Won't this permanent cache make it harder for provisioner to discover newly mounted volumes? I think in 1.9 we will finally get mount propagation which will allow provisioner to discover new mount points on the fly and without a container restart.

Reply:
This PR isn't caching them across multiple runs, but within a single call of discoverVolumesAtPath. Instead of gathering the mountpoints when checking every mountpoint. I'll modify the text, and remove the word Cache.

Reply:
Ah, my bad.